pyccl/ccl_sigM.i

```diff
@@ -0,0 +1,101 @@
+%module ccl_massfunc
+
+%{
+/* put additional #include here */
+%}
+
+%include "../include/ccl_massfunc.h"
+
+// Enable vectorised arguments for arrays
+%apply (double* IN_ARRAY1, int DIM1) {(double* logM, int nM)};
+%apply (int DIM1, double* ARGOUT_ARRAY1) {(int nout, double* output)};
+
+/* The python code here will be executed before all of the functions that
+   follow this directive. */
+%feature("pythonprepend") %{
+    if numpy.shape(logM) != (nout,):
+        raise CCLError("Input shape for `halo_mass` must match `(nout,)`!")
+%}
+
+%inline %{
+
+void sigM_vec(ccl_cosmology * cosmo, double a,
+	      double *logM, int nM,
+	      int nout, double* output, int *status)
+{
+  if (cosmo->params.N_nu_mass>0){
```

Comment:
I think this should be a warning, consistent with how we handle the growth function.

Reply:
These are all actually checked when calling `compute_sigma` from python (which is called within `sigmaM`), so there's no need for them. Removed